### PR TITLE
Publish SA-2018-001

### PR DESCRIPTION
**Pull request by nickvergessen**
https://github.com/nextcloud/security-advisories/pull/1

Script for generating is in https://github.com/nextcloud/security-advisories/blob/master/generator/generate.php

### CODE REVIEW

File: advisories/server-list-part.php [hunk 1]
```diff
@@ -1,4 +1,9 @@
-<p>Version 11.0.3</p>
+<p>Version 12.0.5</p>
+<a href="/security/advisory/?id=nc-sa-2018-001">App password scope can be changed for other users</a><br>
+<br/><p>Version 11.0.7</p>
+<a href="/security/advisory/?id=nc-sa-2018-001">App password scope can be changed for other users</a><br>
+<br/><p>Version 11.0.3</p>
+<a href="/security/advisory/?id=nc-sa-2017-009">Limitation of app specific password scope can be bypassed</a><br>
```
Comment (MorrisJobke) [inline advisories/server-list-part.php]: Duplicate of line 9?

File: advisories/advisory-side.php [hunk 2]
```diff
@@ -1,6 +1,2 @@
-<br/><p>Nextcloud server 11.0.3</p>
-<a href="/security/advisory/?id=nc-sa-2017-007">DOM XSS vulnerability in search dialogue</a><br/>
-<a href="/security/advisory/?id=nc-sa-2017-008">Reflected XSS in error pages</a><br/>
-<a href="/security/advisory/?id=nc-sa-2017-009">Limitation of app specific password scope can be bypassed</a><br/>
-<a href="/security/advisory/?id=nc-sa-2017-010">Stored XSS in Gallery application</a><br/>
-<a href="/security/advisory/?id=nc-sa-2017-011">Share tokens for public calendars disclosed</a><br/>
+<br/><p>Nextcloud server 12.0.5</p>
+<a href="/security/advisory/?id=nc-sa-2018-001">App password scope can be changed for other users</a><br/>
```
Comment (MorrisJobke) [inline advisories/advisory-side.php]: What is this page for?

Comment (MorrisJobke) [reply advisories/advisory-side.php]: This seems to be the sidebar in https://nextcloud.com/security/advisory/?id=nc-sa-2017-010 ... kind of weird that every page has the same side bar. :/

Comment (nickvergessen) [reply advisories/advisory-side.php]: https://next.spreed.me/call/102636166

File: advisories/server-list-part.php (hunk 1 above)
Comment (nickvergessen) [reply advisories/server-list-part.php]: Right, I resorted them because they were not sorted anymore after the generation. Fixed it.

**Comment by jospoortvliet** [comment]
Tested, WorksForMe™

**Pull request merged**